icon/parse

```diff
@@ -43,9 +43,9 @@ OMNI_LATEST=false
 CONTINUE_TEST=false
 
 # ICON related variables
-ICON_MAIN_REPO="git@github.com:C2SM-RCM/icon-eniac.git"
-ICON_REP="icon-eniac"
-STANDALONE_GIT_BRANCH=""
+ICON_MAIN_REPO="git@gitlab.dkrz.de:icon/icon.git"
+ICON_REP="icon-clone"
+STANDALONE_GIT_BRANCH="icon-2.6.2-rc"
```

Comment:
is this just a default or the actual used value. If it is a defautl I would set master.
In anycase we should be able to change this in the Jenkins plan (otherwise we will have to update claw every time we change the rc number)